src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        boolean result = false;
+        if (userService.findByEmail(email) != null
+                && (password.equals(userService.findByEmail(email).getPassword()))) {
+            return true;
+        } else {
+            result = false;
+        }
```

Comment:
two points not fixed - you should call findByEmail only once, and you don't need to create a local boolean variable, you can return directly